python/triton/language/semantic.py

```diff
@@ -1289,7 +1289,16 @@ def atomic_xchg(ptr: tl.tensor, val: tl.tensor, mask: tl.tensor, sem: str, scope
 # ===----------------------------------------------------------------------===//
 
 
-def dot(lhs: tl.tensor, rhs: tl.tensor, acc: tl.tensor, allow_tf32: bool, max_num_imprecise_acc: int,
+def _str_to_dot_f32_backend(f32_backend, builder):
+    assert f32_backend.lower(
+    ) in builder.options.allowed_dot_f32_backends, f"f32 _backend must be one of {builder.options.allowed_dot_f32_backends}. Got {f32_backend}"
```

Comment:
yapf insists we should format this long line like this... lol

Reply:
I just used a `\`. I opened an issue at https://github.com/google/yapf/issues/1207, but the whole yapf repo doesn't look particularly active...